src/platform/imx8/imx8.x.in

```diff
@@ -148,7 +148,8 @@ _memmap_cacheattr_bp_strict = 0x22F22FFF;
 _memmap_cacheattr_wb_allvalid = 0x44224222;
 _memmap_cacheattr_wt_allvalid = 0x11221222;
 _memmap_cacheattr_bp_allvalid = 0x22222222;
-PROVIDE(_memmap_cacheattr_reset = _memmap_cacheattr_wbna_trapnull);
+_memmap_cacheattr_imx8_wt_allvalid = 0x22212222;
```

Comment:
Don't you want to set it to 4 to also store writes in cache?
I think it would improve the performance even better and with write-through I don't think you need dcache_writeback.

Reply:
Actually thought of the same right now, I'll first try to see if it works and if so will push it as such.

Reply:
Apparently with 4 there it's failing to probe (-EIO), so I cannot push as such. I do agree it would have improved performance more but something makes it fail to probe. So no, won't do the change, not for now at least.

Reply:
What exactly fails? Something inside FW or it doesn't boot at all?

Reply:
I think I also need to make a dcache_writeback on the MU? (the mailbox is ALSO in that region, around 0x92400000+a small amount). Trying that too.

Reply:
Not sure what fails, but I don't have access to etrace or other logging facilities. And I'm not planning on bringing back the serial console based __dsp_printf in this one for the moment.

Reply:
@paulstelian97 @tlauda this is hard to investigate for now. I would prefer to merge it like this and an issue to further investigate the problem later as you are kind of stretched with time.